build/template-Build-run-tests-sign.yml

```diff
@@ -0,0 +1,213 @@
+    projects: '$(WilsonSourceDirectory)Product.proj'
+    arguments: '/r:True /p:Configuration=$(BuildConfiguration) /p:Platform="Any CPU" /verbosity:m /p:SourceLinkCreate=true'
+
+- task: PowerShell@2
+  displayName: 'Run Tests'
+  inputs:
+    targetType: filePath
+    filePath: ./$(WilsonSourceDirectory)runTests.ps1
+    arguments: '-buildType $(BuildConfiguration)'
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: [staging]\ProductBinaries'
+  inputs:
+    SourceFolder: $(WilsonSourceDirectory)src
+    Contents: |
+      **\bin\$(BuildConfiguration)\**\Microsoft.IdentityModel.*.dll
+      **\bin\$(BuildConfiguration)\**\Microsoft.IdentityModel.*.pdb
+      **\bin\$(BuildConfiguration)\**\System.IdentityModel.Tokens.Jwt.dll
+      **\bin\$(BuildConfiguration)\**\System.IdentityModel.Tokens.Jwt.pdb
+    TargetFolder: '$(Build.ArtifactStagingDirectory)\ProductBinaries'
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+  displayName: 'Run PoliCheck'
+  inputs:
+    targetType: F
+    result: PoliCheck.xml
+    optionsFC: 0
+    optionsXS: 0
+    optionsHMENABLE: 0
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
+  displayName: 'Run CredScan'
+  inputs:
+    outputFormat: pre
+    suppressionsFile: 'build/credscan-exclusion.json'
+    debugMode: false
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-roslynanalyzers.RoslynAnalyzers@2
+  displayName: 'Run Roslyn Analyzers'
+  condition: and(eq(variables['TargetNet8'], 'False'), eq(variables['PipelineType'], 'Legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@4
+  displayName: 'Run BinSkim'
+  inputs:
+    AnalyzeSymPath: '$(Build.ArtifactStagingDirectory)\ProductBinaries'
+    AnalyzeVerbose: true
+    AnalyzeHashes: true
+  condition: and(succeeded(), eq(variables['PipelineType'], 'legacy'))
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+  displayName: 'Publish Security Analysis Logs'
+  continueOnError: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+  displayName: 'Post SDL Analysis'
+  continueOnError: true
+
+#Sign Wilson 7x task group
+- template: template-sign-wilson.yaml
+
+- task: DotNetCoreCLI@2
+  displayName: 'Pack libraries'
+  inputs:
+    command: 'pack'
+    arguments: '--no-restore'
+    packDirectory: '$(Build.SourcesDirectory)\artifacts'
+    nobuild: true
+    verbosityPack: 'Minimal'
+    configuration: $(BuildConfiguration)
+    packagesToPack: '$(WilsonSourceDirectory)Product.proj'
+    feedsToUse: 'config'
+    nugetConfigPath: Nuget.config
+    externalFeedCredentials: 'Internal Analyzers'
+
+- task: EsrpCodeSigning@4
```

Comment:
Is this still needed if ESRP code signing is still on?

Reply:
Not necessarily, I am just keeping the old build functional until we finally pull the plug on it.

Reply:
Got it, I see the "legacy" flag now, thanks. Maybe add a comment.